pfcpiface/parse-pdr.go

```diff
@@ -47,33 +49,34 @@ func needAllocIP(ueIPaddr *ie.UEIPAddressFields) bool {
 	return true
 }
 
-func (p *pdr) printPDR() {
-	log.Println("------------------ PDR ---------------------")
-	log.Println("Src Iface:", p.srcIface)
-	log.Println("tunnelIP4Dst:", int2ip(p.tunnelIP4Dst))
-	log.Println("tunnelTEID:", p.tunnelTEID)
-	log.Println("srcIP:", int2ip(p.srcIP))
-	log.Println("dstIP:", int2ip(p.dstIP))
-	log.Println("srcPort:", p.srcPort)
-	log.Println("dstPort:", p.dstPort)
-	log.Println("proto:", p.proto)
-	log.Println("Src Iface Mask:", p.srcIfaceMask)
-	log.Println("tunnelIP4Dst Mask:", int2ip(p.tunnelIP4DstMask))
-	log.Println("tunnelTEIDMask Mask:", p.tunnelTEIDMask)
-	log.Println("srcIP Mask:", int2ip(p.srcIPMask))
-	log.Println("dstIP Mask:", int2ip(p.dstIPMask))
-	log.Println("srcPort Mask:", p.srcPortMask)
-	log.Println("dstPort Mask:", p.dstPortMask)
-	log.Println("proto Mask:", p.protoMask)
-	log.Println("pdrID:", p.pdrID)
-	log.Println("fseID", p.fseID)
-	log.Println("fseidIP", p.fseidIP)
-	log.Println("ctrID:", p.ctrID)
-	log.Println("farID:", p.farID)
-	log.Println("qerID:", p.qerID)
-	log.Println("needDecap:", p.needDecap)
-	log.Println("allocIPFlag:", p.allocIPFlag)
-	log.Println("--------------------------------------------")
+// Satisfies the fmt.Stringer interface
+func (p pdr) String() string {
+	var b = strings.Builder{}
+	fmt.Println(&b, "srcIface:", p.srcIface)
+	fmt.Println(&b, "tunnelIP4Dst:", int2ip(p.tunnelIP4Dst))
+	fmt.Println(&b, "tunnelTEID:", p.tunnelTEID)
+	fmt.Println(&b, "srcIP:", int2ip(p.srcIP))
+	fmt.Println(&b, "dstIP:", int2ip(p.dstIP))
+	fmt.Println(&b, "srcPort:", p.srcPort)
+	fmt.Println(&b, "dstPort:", p.dstPort)
+	fmt.Println(&b, "proto:", p.proto)
+	fmt.Println(&b, "srcIfaceMask:", p.srcIfaceMask)
```

Comment:
```suggestion
	fmt.Fprintf(&b, "srcIfaceMask:%#x\n", p.srcIfaceMask)
```
Similar for all non ip subnet masks